WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAccountPendingView.swift

```diff
@@ -0,0 +1,67 @@
+import SwiftUI
+
+struct InPersonPaymentsStripeAccountPending: View {
+    let deadline: Date?
+
+    var body: some View {
+          VStack {
+              Spacer()
+
+              VStack(alignment: .center, spacing: 42) {
+                  Text(Localization.title)
+                      .font(.headline)
+                  Image(uiImage: .paymentErrorImage)
+                      .resizable()
+                      .scaledToFit()
+                      .frame(height: 180.0)
+                  Text(message)
+                      .font(.callout)
+                  InPersonPaymentsSupportLink()
+              }
+              .multilineTextAlignment(.center)
+
+              Spacer()
+
+              InPersonPaymentsLearnMore()
+          }
+          .padding(24.0)
+      }
+
+    private var message: String {
+        guard let deadline = deadline else {
+            DDLogError("In-Person Payments not avilable. Stripe has pending requirements without known deadline")
+            return Localization.messageUnknownDeadline
+        }
+        return String(format: Localization.messageDeadline, deadline.toString(dateStyle: .medium, timeStyle: .none))
+    }
+}
+
+private enum Localization {
+    static let title = NSLocalizedString(
+        "Your WooCommerce Payments account has pending requirements",
+        comment: "Title for the error screen when the Stripe account is restricted because there are pending requirements"
+    )
+
+    static let messageDeadline = NSLocalizedString(
+        "There are pending requirements in your account. Please complete those requirements by %1$@ to keep accepting in-Person Payments.",
+        comment: "Error message when WooCommerce Payments is not supported because there are pending requirements in the Stripe account (with a known deadline)"
```

Comment:
I'd suggest specifying that `%1$@` represents a date with an example of the format in English:

```suggestion
        comment: "Error message when WooCommerce Payments is not supported because there are pending requirements in the Stripe account. %1$d will contain the localized deadline (e.g. August 11, 2021)"
```